### PR TITLE
[FEATURE] Add workflow demo

### PR DESCRIPTION
**Pull request by klauste**
LRN-28051

### CODE REVIEW

File: www/authoring/index.php [hunk 1]
```diff
@@ -173,6 +173,22 @@
...
+                    <div class="panel-body">
+                        <p>Move your item through a series of workflow states, from a first draft to the
+                            finished product.</p>
+                        <p class="text-right">
+                            <a class="demo_link"  href="./workflow.php">Demo</a>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
```
Comment (hbalseiro) [inline www/authoring/index.php:191]: @klauste I'd suggest a minor change: moving this down to the "Creating Content, Questions, and Features" section just below for clarity. Right now it's in the "Browsing Items and Activities" section which seems better suited to item list/activity list related functionality.

Comment (klauste) [reply www/authoring/index.php]: @hbalseiro I've moved the workflow demo as you suggested. Does it matter that the demo opens the item list page (that's on purpose as searching by workflow is possible)?

**Comment by PhilBroadbent** [comment]
@klauste looks good, only thing I'd suggest is adding a line about init options, as other demos do that:

` For more information refer to the init options docs .` with a link there

**Comment by klauste** [comment]
@PhilBroadbent I've added a link to the item workflow init option reference.

File: www/authoring/index.php (hunk 1 above)
Comment (hbalseiro) [reply www/authoring/index.php]: @klauste No, that should work just fine. To be honest, we wouldn't object if Team Authoring felt the other section was the best place for it, it was sincerely just a suggestion. We're mainly concerned with keeping the code simple and readable so clients can understand it at a glance and yours was perfect in that sense.

**Pull request merged**